.test-infra/jenkins/LoadTestsBuilder.groovy

```diff
@@ -18,26 +18,33 @@
 
 import CommonJobProperties as commonJobProperties
 import CommonTestProperties.Runner
+import CommonTestProperties.SDK
 
 class LoadTestsBuilder {
+    static void loadTest(context, String title, Runner runner, SDK sdk, Map<String, ?> options, String mainClass) {
+        String task
+        if (sdk == SKD.JAVA) {
+            task = ':beam-sdks-java-load-tests:run'
+        } else if (sdk == SKD.PYTHON) {
+            task = ':beam-sdks-python-load-tests:run'
+        }
```

Comment:
@lgajowy I changed it, so there are no 2 methods per SDK. I didn't use `elvis` operator to be precise about chosen sdk.